inst/rmarkdown/templates/jss_article/skeleton/skeleton.Rmd

```diff
@@ -3,9 +3,9 @@ documentclass: jss
 author:
   - name: FirstName LastName
     affiliation: University/Company
-    address: >
-      First line
-      Second line
+    address: |
+      | First line
+      | Second line
```

Comment:
Are the leading pipes necessary? Probably not. Could you [double check](https://pandoc.org/MANUAL.html#metadata-blocks)?

Reply:
I blindly commit the suggestion  in #254, but I'll check that.

Reply:
I just checked it. They are necessary. The meanings of the pipe after and below `address:` are different. The former is explained at https://pandoc.org/MANUAL.html#metadata-blocks and the latter means https://pandoc.org/MANUAL.html#line-blocks

Reply:
Thanks ! That is good to know!